apps/worker/tests/test_process_job.py

```
@@ -411,6 +439,142 @@ def test_process_job_marks_unknown_job_type_failed_without_calling_github():
+    conn = _FakeConn()
+
+    resp = MagicMock()
+    resp.status_code = 403
+    resp.text = ""
+    resp.headers = {"Retry-After": "1"}
+
+    with patch("worker.httpx.Client") as mock_client_cls:
+        mock_client_cls.return_value = _mock_client(delete=MagicMock(return_value=resp))
+        process_job(conn, 27, "github.clear_actions_cache", _payload(), retry_count=0)
+
+    sql, params = conn._cursor.calls[0]
```

Ruff (0.16.3)

[warning] Unpacked variable `params` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)